mlrun/runtimes/sparkjob/abstract.py

```diff
@@ -100,6 +100,8 @@ class AbstractSparkJobSpec(KubeResourceSpec):
         "main_class",
         "spark_conf",
         "hadoop_conf",
+        "driver_java_options",
+        "executor_java_options",
```

Comment:
If you're only adding these in the Spark-3 case, why not place these parameters in `Spark3JobSpec` rather than in `AbstractSparkJobSpec`?